augur/application/cli/backend.py

```diff
@@ -187,13 +187,15 @@ def augur_stop(signal, logger):
     """
 
     augur_processes = get_augur_processes()
-    _broadcast_signal_to_processes(augur_processes, broadcast_signal=signal, given_logger=logger)
-
     # if celery is running, run the cleanup function
     process_names = [process.name() for process in augur_processes]
+
+    _broadcast_signal_to_processes(augur_processes, broadcast_signal=signal, given_logger=logger)
+
     if "celery" in process_names:
```

Comment:
It was throwing errors when it was in the old order. I think this was because when the processes were being killed it was clearing the linux stat file which is what the process name was coming from. It was erroring almost every time before for me.

Reply:
That makes sense. I don't think I ever ran collection after making that change